src_kt/java/arcs/Arc.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs
+
+class Arc {
+  // TODO: everything(?)
```

Comment:
Love it :)

Out of interest, what depends on an arc object? I'm guessing it's the volatile store? We might be able to refactor that dependency to just be an ID, which would simplify things.

Reply:
Yeah I made this file because at first glance I saw VolatileDriver required it, but I managed to refactor to only need `ArcId`.